cfgov/jinja2/v1/_includes/organisms/main-contact-info.html

```diff
@@ -0,0 +1,73 @@
+{# ==========================================================================
+
+   main_contact_info.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create a Main Contact Information organism.
+   See [GHE]/flapjack/Modules-V1/wiki/Contact-Information-(Main-Content-Area)
+
+   content:                  An object containing sidebar content.
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/97dc5969011f19f33f1f0a38f8d4a14065e9b554